.circleci/config.yml

```diff
@@ -140,15 +211,89 @@ jobs:
       - run:
           name: Run test vectors
           command: make run-vectors
-  install:
+  coverage:
+    executor: mac-executor
```

Comment:
this should maybe use the linux executor? or is this intentional?

Reply:
This is intentional, I kept getting unexpected dependency issues while running in the linux executor. I plan to investigate this more but for now should be sufficient.

Reply:
Maybe add a TODO comment to indicate this, so it doesn't get lost in the future? Or some mechanism to come back to this?